react/tic-tac-toe/src/app/screens/Game/index.js

```diff
@@ -1,20 +1,99 @@
 import React from 'react';
-
-import styles from './styles.module.scss';
 import Board from './components/Board';
 
-function Game () {
-  return (
-    <div className={styles.game}>
-      <div className={styles.gameBoard}>
-        <Board />
-      </div>
-      <div className={styles.gameInfo}>
-        <div>{/* status */}</div>
-        <ol>{/* TODO */}</ol>
+class Game extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      history: [{
+        squares: Array(9).fill(null),
+      }],
+      stepNumber: 0,
+      xIsNext: true,
+    };
+  }
+
+  handleClick(i) {
+    const history = this.state.history.slice(0, this.state.stepNumber + 1);
```

Comment:
destruct state first, for example `const { history } = this.state;` To avoid repeat this.state each time you refear to state.